src/components/FilterSitesCountText.js

```diff
@@ -0,0 +1,135 @@
+import PropTypes from 'prop-types'
+import React, { useState, useEffect } from 'react'
+
+import styled from 'styled-components'
+import { getSitesGroupBySampleEvents } from '../lib/array-helpers'
+import { dropdownOptionsPropType, sitesPropType } from '../lib/mermaidDataPropTypes'
+
+const FilteredText = styled('div')`
+  padding-bottom: 10px;
+`
+
+const FilterSitesCountText = ({
+  allSites,
+  countryOptionValues,
+  projectOptionValues,
+  organizationOptionValues,
+  startDate,
+  endDate,
+}) => {
+  const [filteredSitesCount, setFilteredSitesCount] = useState(0)
+
+  const _updateFilteredSitesCount = useEffect(() => {
+    const sampleEvents = allSites.map(site => site[1]).flat()
+    const startDateTime = new Date(startDate).getTime()
+    const endDateTime = new Date(endDate).getTime()
+    const countryOptionLabels = countryOptionValues.map(({ label }) => label)
+    const projectOptionLabels = projectOptionValues.map(({ label }) => label)
+    const organizationOptionLabels = organizationOptionValues.map(({ label }) => label)
+
+    const allFiltersSelected =
+      countryOptionLabels.length && projectOptionLabels.length && organizationOptionLabels.length
+    const oneOfFiltersSelected =
+      countryOptionLabels.length || projectOptionLabels.length || organizationOptionLabels.length
+    const noneOfFiltersSelected =
+      !countryOptionLabels.length && !projectOptionLabels.length && !organizationOptionLabels.length
+    const countryAndProjectFiltersSelected =
+      countryOptionLabels.length && projectOptionLabels.length && !organizationOptionLabels.length
+    const countryAndOrganizationFiltersSelected =
+      countryOptionLabels.length && !projectOptionLabels.length && organizationOptionLabels.length
+    const projectAndOrganizationFiltersSelected =
+      !countryOptionLabels.length && projectOptionLabels.length && organizationOptionLabels.length
```

Comment:
these are all boolean vars. Can you name them with an `is` or `are` prefix. Eg `areAllFiltersSelected`, `isAnyFilterSelected`

Reply:
nice that they are named though 👍

Reply:
make sense